src/schema/sale_artwork.js

```diff
@@ -60,9 +61,7 @@ const SaleArtworkType = new GraphQLObjectType({
               // tiers to avoid mutating the cached value.
               const tiers = incrs[0].increments.slice()
               const increments = [minimum_next_bid_cents]
-              const limit =
-                BIDDER_POSITION_MAX_BID_AMOUNT_CENTS_LIMIT ||
-                Number.MAX_SAFE_INTEGER
+              const limit = BIDDER_POSITION_MAX_BID_AMOUNT_CENTS_LIMIT
```

Comment:
#minor probably don't need `const limit` anymore since you moved the default of `Number.MAX_SAFE_INTEGER` up to `config`